src/main/java/com.amazon.bookstore/BookstoreController.java

```diff
@@ -7,8 +7,8 @@
 @Controller
 public class BookstoreController {
 
-    @GetMapping("/")
-    public String index(Model model) {
```

Comment:
If this index method is removed, we have to create an index.html page that our program goes to when there is an error (a fallback page).

Reply:
What if I make homePage() the index() method instead? That way it falls back to the very beginning

Reply:
That sounds like it should work